index.html

```diff
@@ -600,10 +600,10 @@ <h3>Presentations</h3>
 specified interval. Selective disclosure schemes using zero-knowledge proofs
 can use <a>claims</a> expressed in this model to prove additional statements
 about those <a>claims</a>. For example, a <a>claim</a> specifying a
-<a>subject</a>'s date of birth can be used as a predicate to prove that the
-<a>subject</a>'s age is within a given range, and therefore prove the
+<a>subject's</a> date of birth can be used as a predicate to prove the
```

Comment:
This is going to make the auto-linker fail. I'll merge it in anyway and add an alias.